packages/shared/forks/ReactFeatureFlags.native-fb.js

```diff
@@ -37,7 +37,6 @@ export const disableJavaScriptURLs = false;
 export const disableCommentsAsDOMContainers = true;
```

Comment:
@alunyov: Dynamic flags would be imported from some module like `ReactNativeInternalFeatureFlags` here on line 16 (can't comment on lines above??